cnames_active.js

```diff
@@ -1925,6 +1925,7 @@ var cnames_active = {
   "md": "cname-china.vercel-dns.com", // noCF
   "md5": "cname-china.vercel-dns.com", // noCF
   "mdbf-css": "more-beautiful-div-framework.github.io",
+  "mdxe": "vercel-dns.com", // noCF
```

Comment:
```suggestion
  "mdxe": "cname.vercel-dns.com", // noCF
```
👋 For Vercel content, you'll want to target `cname.vercel-dns.com`.

Also, as the CI notes, your entry needs to be in alphabetical order, please